website/contact/index.html

```diff
@@ -57,6 +57,22 @@ <h3>{{ _('<a href="irc://irc.mozilla.org/thunderbird">Live Chat on IRC</a>') }}<
     </div>
   </section><!-- end #support -->
 
+  <section id="security">
+    <h2>{{ _('Security') }}</h2>
+    <div class="row">
+      <div class="col full">
+        <p>{{ _('For developers who would like to report security issues.') }}</p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col full">
+        <section>
+          <p>{{ _('To report a security issue related to Thunderbird, please register an account at <a href="https://bugzilla.mozilla.org/">Bugzilla</a>, and describe the issue in a new Thunderbird <a href="https://bugzilla.mozilla.org/enter_bug.cgi">bug report</a>. If you select the <i>Security checkmark</i> before you submit, then only members of our security team will have access to your report. You may also contact us at <a href="mailto:security@thunderbird.net">security@thunderbird.net</a>. If necessary, email encryption with members of our team can be arranged.') }}</p>
```

Comment:
Rather than linking to the generic bug submission page, I think the text "bug report" should include the Thunderbird product: https://bugzilla.mozilla.org/enter_bug.cgi?product=Thunderbird